<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevSetting_Mic</class>
 <widget class="QWidget" name="DevSetting_Mic">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AMP Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:checked{
	background-color: rgb(0, 170, 0);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xover">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Xover</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dyn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dyn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eq">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Eq</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Gain">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Phantom">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Phantom</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>33</width>
     <height>23</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>N1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="VolumeWidget" name="Meter_AIn_1" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>33</width>
     <height>121</height>
    </rect>
   </property>
   <property name="MaxValue" stdset="0">
    <number>0</number>
   </property>
   <property name="MinValue" stdset="0">
    <number>-70</number>
   </property>
   <property name="SingleTrack" stdset="0">
    <bool>true</bool>
   </property>
   <property name="ScaleLabel" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="VolumeWidget" name="Meter_NOut_1" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>33</width>
     <height>121</height>
    </rect>
   </property>
   <property name="MaxValue" stdset="0">
    <number>0</number>
   </property>
   <property name="MinValue" stdset="0">
    <number>-70</number>
   </property>
   <property name="SingleTrack" stdset="0">
    <bool>true</bool>
   </property>
   <property name="ScaleLabel" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>33</width>
     <height>23</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>A1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VolumeWidget</class>
   <extends>QWidget</extends>
   <header>customWidget/volumewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
